.gitmodules

```diff
@@ -12,6 +12,9 @@
     path = modules/chemical_reactions/contrib/thermochimica
     url = https://github.com/ORNL-CEES/thermochimica.git
     update = none
+[submodule "modules/fluid_properties/contrib/saline"]
+	path = modules/fluid_properties/contrib/saline
+	url = https://code.ornl.gov/neams/saline
```

Comment:
I don't know if we want this checked out by default?

Reply:
probably safe not to.  Just set `update = none`?

Reply:
yea